src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Autoplay, A11y } from 'swiper/modules';
+import ProductCard from '../ProductCard/ProductCard';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import './Carousel.scss';
+import NextBtn from './NextBtn';
+import PrevBtn from './PrevBtn';
+
+const Carousel = ({ slideList }) => {
+  return (
+    <div className="carousel">
+      <Swiper
+        modules={[Autoplay, A11y]}
+        spaceBetween={10}
+        slidesPerView={4}
+        // loop={true}
+        autoplay={{
+          delay: 3000,
+          disableOnInteraction: false,
+        }}
+      >
+        {slideList &&
+          slideList.map(list => (
```

Comment:
props로 전달하는 state의 초기값이 빈 배열인데, 빈 배열은 truthy한 값이기 때문에 조건은 항상 true겠네요!
```suggestion
        {slideList.map(list => (
```